db/migrate/20150918094007_create_discussions_and_forums.rb

```diff
@@ -0,0 +1,65 @@
+class CreateDiscussionsAndForums < ActiveRecord::Migration
+  def change
```

Comment:
Assignment Branch Condition size for change is too high. [41/15]<br>Method has too many lines. [47/10]